core/src/test/scala/integration/kafka/api/BaseAdminIntegrationTest.scala

```diff
@@ -30,9 +30,9 @@ import org.apache.kafka.common.resource.ResourceType
 import org.apache.kafka.common.utils.Utils
 import org.apache.kafka.coordinator.group.GroupCoordinatorConfig
 import org.apache.kafka.security.authorizer.AclEntry
-import org.apache.kafka.server.config.{ServerConfigs, ReplicationConfigs, ServerLogConfigs}
+import org.apache.kafka.server.config.{ReplicationConfigs, ServerConfigs, ServerLogConfigs}
```

Comment:
can these changes be reverted?

Reply:
Ah, intellj's smart formatting... Updated.